inc/Integrations/Google/Sheets/GoogleSheetsDatasource.php

```diff
@@ -46,31 +48,22 @@ class GoogleSheetsDatasource extends HttpDatasource {
 					'universe_domain'             => [ 'type' => 'string' ],
 				],
 			],
+			'display_name'   => [ 'type' => 'string' ],
```

Comment:
@mhsdef Is there a reason we made `display_name` mandatory here, but are not requiring it for other data sources ?

Reply:
probably accidental that it alone is that way. I'll adjust it to optional too.

I think, ultimately, we'll want to require on the backend. Maybe able to prefill it (or the slug?) on the front for better UX.